test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -136,19 +136,19 @@
              (merge valid-ssl-pem-config
                     {:ssl-host "foo.local"}))
            (munge-expected-https-config
-             {:https {:host "foo.local" :port default-https-port}})))
+             {:https {:host "foo.local" :port default-https-port :disallow-renegotiation default-disallow-renegotiation}})))
```

Comment:
This map is now long enough that I think it makes sense to break it up by key-value pairs.